src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -640,7 +747,6 @@ private void PrepareSampleProjectsAndConfig(
         _env.SetCurrentDirectory(Path.GetDirectoryName(projectFile.Path));
 
         _env.SetEnvironmentVariable("MSBUILDNOINPROCNODE", buildInOutOfProcessNode ? "1" : "0");
-        _env.SetEnvironmentVariable("MSBUILDLOGPROPERTIESANDITEMSAFTEREVALUATION", "1");
```

Comment:
what does `MSBUILDLOGPROPERTIESANDITEMSAFTEREVALUATION` do?

Reply:
This ensures that evaluated properties and items are sent back to the main node. This is not default behavior.
The BuildCheck forwarding and central loggers now make sure this is happening - the tests do not need to request that explicitly